schema.cue

```diff
@@ -0,0 +1,62 @@
+import (
+	"time"
+)
+
+// Date is a date in the format YYYY-MM-DD
+#Timestamp: time.Format(time.RFC3339)
+
+#TypeURI: string
+
+#BaselineControlID: =~"^OSPS-[A-Z]{2}-[0-9]{2}.[0-9]{2}$"
+
+#ResourceDescriptor: {
+	description: string
+	mediaType:   string
+	uri:         #TypeURI
+}
+
+#Ella: {
+	uri:     #TypeURI
```

Comment:
Whoops, this should be a ResourceURI: https://github.com/in-toto/attestation/blob/main/spec/v1/field_types.md#resourceuri

Corrected in the source doc.